subworkflows/local/starfusion_workflow.nf

```diff
@@ -51,8 +51,9 @@ workflow STARFUSION_WORKFLOW {
     emit:
         fusions               = ch_starfusion_fusions
         star_stats            = ch_star_stats
-        bam_sorted            = ch_align
+        ch_bam_sorted         = ch_align.ifEmpty([[],[]])
+        ch_bam_sorted_indexed = bam_sorted_indexed.ifEmpty([[],[],[]])
```

Comment:
This are the meta, bam, bai elements that the channel contains. In case starfusion is not a selected tool, the channels need to contain empty information in the proper format, hence  []

Reply:
thanks! now I know :)